src/cmd/shared.go

```diff
@@ -47,6 +51,46 @@ func getAwsConfig() (types.AwsConfig, error) {
 	}, nil
 }
 
+func getDeployConfig() (types.DeployConfig, error) {
+	appDir, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+	homeDir, err := util.GetHomeDirectory()
+	if err != nil {
+		panic(err)
```

Comment:
Why are we panic-ing instead of returning the error since this function returns an error?